packages/tasks/src/tasks/text-to-image/data.ts

```diff
@@ -93,7 +93,7 @@ const taskData: TaskDataCustom = {
 	],
 	summary:
 		"Generates images from input text. These models can be used to generate and modify images based on text prompts.",
-	widgetModels: ["CompVis/stable-diffusion-v1-4"],
+	widgetModels: ["black-forest-labs/FLUX.1-dev"],
```

Comment:
I would also add flux to the list of models

Reply:
I think it already exists no? on the first row

Reply:
Yes you're right!